packages/portal/swagger-ui-web-component/src/element.js

```diff
@@ -223,9 +223,22 @@ export class SwaggerUIElement extends HTMLElement {
     }
 
     const operationElementId = operationToSwaggerThingId(operation)
-    const element = this.shadowRoot.getElementById(operationElementId)
+    let element = this.shadowRoot.getElementById(operationElementId)
```

Comment:
suggestion: if it doesn’t have a huge blast radius, could you rename the `element` var to something that isn’t a primitive? Not a huge deal